integrations/etcd/documentation.yaml

```diff
@@ -0,0 +1,83 @@
+exporter_type: included
+app_name_short: Etcd
+app_name: {{app_name_short}}
+app_site_name: Etcd
+app_site_url: https://etcd.io/
+exporter_name: the Etcd exporter
+exporter_pkg_name: etcd
+exporter_repo_url: https://etcd.io/docs/v3.1/op-guide/monitoring/
+additional_prereq_info: |
+  {{app_site_name}} exposes Prometheus-format metrics automatically; you do not
+  have to install it separately. To verify that {{exporter_name}} is emitting 
+  metrics on the expected endpoints, do the following:
+
+  1. Set up port forwarding by using the following command:
+
+  <pre class="devsite-click-to-copy">
+  kubectl -n {{namespace_name}} port-forward {{pod_name}} 2379
+  </pre>
+
+  2. Access the endpoint `localhost:2379/metrics` by using the browser
+  or the `curl` utility in another terminal session.
+dashboard_available: true
+multiple_dashboards: false
+dashboard_display_name: {{app_name_short}} Prometheus Overview
+podmonitoring_config: |
+  apiVersion: monitoring.googleapis.com/v1
+  kind: PodMonitoring
+  metadata:
+    name: etcd
+    labels:
+      app.kubernetes.io/name: etcd
+      app.kubernetes.io/part-of: google-cloud-managed-prometheus
+  spec:
+    endpoints:
+    - port: 2379
+      scheme: http
+      interval: 30s
+      path: /metrics
+    selector:
+      matchLabels:
+        app.kubernetes.io/name: etcd
+additional_podmonitoring_info: |
+  Ensure that the values of the `port` and `matchLabels` fields match those of the {{app_name_short}} pods you want to monitor.
+  Deploying {{app_name_short}} using this [example](https://github.com/etcd-io/etcd/tree/main/hack/kubernetes-deploy){:class=external}
+  will contain the label `app: etcd` and `client` port.
```

Comment:
One last thing - if this text says "Deploying etcd using this example will contain the label `app: etcd`", we should use `app: etcd` in the podmonitoring config (line 41 above)? Because this paragraph is explaining why the label selectors in the sample podmonitoring config we provide are what they are.

Or should we keep line 41 unchanged, and change this paragraph to match? This was just one example I found, but not sure if there are more common helm charts or installation methods out there that do label your pods with `app.kubernetes.io/name: etcd` by default?

Reply:
There is an existing helm chart packaged by Bitnami that will label the pods that way by default, I've updated the example to refer to that instead.